x-pack/plugins/security_solution/public/flyout/entity_details/host_right/index.tsx

```diff
@@ -101,43 +98,18 @@ export const HostPanel = ({
     { onSuccess: refetchRiskScore }
   );
 
-  const { data } = useMisconfigurationPreview({
-    query: buildEntityFlyoutPreviewQuery('host.name', hostName),
-    sort: [],
-    enabled: true,
-    pageSize: 1,
-    ignore_unavailable: true,
-  });
-
-  const passedFindings = data?.count.passed || 0;
-  const failedFindings = data?.count.failed || 0;
-
-  const hasMisconfigurationFindings = passedFindings > 0 || failedFindings > 0;
-
-  const { data: vulnerabilitiesData } = useVulnerabilitiesPreview({
-    query: buildEntityFlyoutPreviewQuery('host.name', hostName),
-    sort: [],
-    enabled: true,
-    pageSize: 1,
-  });
+  const { hasMisconfigurationFindings } = useHasMisconfigurations('host.name', hostName);
 
-  const hasVulnerabilitiesFindings = sum(Object.values(vulnerabilitiesData?.count || {})) > 0;
+  const { hasVulnerabilitiesFindings } = useHasVulnerabilities('host.name', hostName);
 
-  const { signalIndexName } = useSignalIndex();
-
-  const entityFilter = useMemo(() => ({ field: 'host.name', value: hostName }), [hostName]);
-
-  const { items: alertsData } = useAlertsByStatus({
-    entityFilter,
-    signalIndexName,
-    queryId: `${DETECTION_RESPONSE_ALERTS_BY_STATUS_ID}HOST_NAME_RIGHT`,
+  const { hasNonClosedAlerts } = useNonClosedAlerts({
+    fieldName: 'host.name',
```

Comment:
I'd put `host.name` in a constant because this literal is passed down to all these hooks